src/pages/patientView/clinicalInformation/ClinicalInformationSamplesTable.tsx

```diff
@@ -64,3 +66,8 @@ export default class ClinicalInformationSamplesTable extends React.Component<ICl
         return tableData;
     }
 }
+
```

Comment:
this is my fault, but lets put this interface with other interface at top of file.  also, lets export it in case it's needed elsewhere.